airflow/operators/python.py

```diff
@@ -253,13 +269,21 @@ def execute(self, context: Context) -> Any:
             self.log.info('Proceeding with downstream tasks...')
             return condition
 
-        self.log.info('Skipping downstream tasks...')
-
-        downstream_tasks = context["task"].get_flat_relatives(upstream=False)
-        self.log.debug("Downstream task_ids %s", downstream_tasks)
+        downstream_tasks = context['task'].get_flat_relatives(upstream=False)
+        self.log.debug("Downstream tasks %s", downstream_tasks)
 
         if downstream_tasks:
-            self.skip(context["dag_run"], context["logical_date"], downstream_tasks)
+            dag_run = context["dag_run"]
+            execution_date = context["ti"].execution_date
```

Comment:
```suggestion
            execution_date = dag_run.execution_date
```

These are the exact same value. (Actually, we should probably refactor `skip()` to make the `execution_date` argument optional, it’s totally useless and should only ever be passed for backward compatibility reasons.